### PR TITLE
Adds garage project

### PR DESCRIPTION
**Pull request by renekounkou**
Please review.

### CODE REVIEW

**Comment by renekounkou** [comment]
Please review

File: garage.com/src/main/java/garage/com/LargeEngine.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package garage.com;
+
+public class LargeEngine implements IEngine {
+
+	public int cylinderCount;
+
+	public void start() {
+
```
Comment (dentaylor) [inline garage.com/src/main/java/garage/com/LargeEngine.java]: RW (remove whitespace)

File: garage.com/src/main/java/garage/com/LargeEngine.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		System.out.println("Engine Started");
+
```
Comment (dentaylor) [inline garage.com/src/main/java/garage/com/LargeEngine.java]: RW (remove whitespace)

File: garage.com/src/test/java/Car.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+public class Car {
+
+	private int odometer;
+	public IEngine engine;
+	public String model;
+
+	public  Car(String model) {
+		
+		this.model = model;
+
```
Comment (dentaylor) [inline garage.com/src/test/java/Car.java]: RW (remove whitespace)

File: garage.com/src/test/java/Car.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+		LargeEngine newEngine = new LargeEngine();
+
```
Comment (dentaylor) [inline garage.com/src/test/java/Car.java]: RW (remove whitespace)

File: garage.com/src/test/java/Car.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+		this.engine = engine;
+
```
Comment (dentaylor) [inline garage.com/src/test/java/Car.java]: RW (remove whitespace)

File: garage.com/src/test/java/Car.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+	}
+
+	public void start() {
+
```
Comment (dentaylor) [inline garage.com/src/test/java/Car.java]: RW (remove whitespace)

File: garage.com/src/test/java/CarTests.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+import org.testng.annotations.Test;
+
+import garage.com.IEngine;
+
+public class CarTests {
+
+
+	@Test
+	public void canBuildCar() {
+
```
Comment (dentaylor) [inline garage.com/src/test/java/CarTests.java]: RW (remove whitespace)

File: garage.com/src/test/java/CarTests.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import garage.com.IEngine;
+
+public class CarTests {
+
```
Comment (dentaylor) [inline garage.com/src/test/java/CarTests.java]: RW (remove whitespace)

File: garage.com/src/test/java/Car.java [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+	}
+
+	public int getOdometer() {
+		return odometer;
```
Comment (dentaylor) [inline garage.com/src/test/java/Car.java]: Use 'this' keyword

Ex: return this.odometer

**Review by dentaylor — CHANGES_REQUESTED** [review]
Resolve the comments then push your changes to the same branch.

**Pull request closed without merging**